conga-aem-definitions/src/main/roles/aem-dispatcher-ams.yaml

```diff
@@ -0,0 +1,518 @@
+      # If set to true always "*" is sent as 'Access-Control-Allow-Origin' header
+      # If set to false only the requesting host is all if it is in the hostWhiteList (or this list is empty), and not in the hostBlackList
+      allowAllHosts: true
+      #hostWhiteList:
+      #- "https://www.host1.com"
+      #- "https://www.host2.com"
+
+    headers:
+      # Enables/Configures the Content-Security-Policy header on publish dispatcher (see https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP)
+      contentSecurityPolicy:
+      # Enables/Configures the Referrer-Policy header on publish dispatcher (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)
+      referrerPolicy: "origin-when-cross-origin"
+      # Enables/Configures the X-XSS-Protection header on publish dispatcher (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection)
+      xssProtection:
+      # Enables/Configures the Cache-Control header on publish dispatcher (see https://developer.mozilla.org/en/docs/Web/HTTP/Headers/Cache-Control)
+      cacheControl:
+        # Cache AEM Clientlibs with long cache key for 1 year
+        - locationMatch: "\\.lc-.*-lc\\.min\\.(css|js)$"
+          directives: "max-age=31536000, public"
+      # Set the default experiation time for text/html responses (enabled by default in AEM Cloud Service webserver)
+      htmlExpirationTimeMin: 5
+
+    # List of rewrite rules to include in the vhost
+    rewriteIncludes:
+      - conf.d/rewrites/rewrite.rules
+
+    # Rewrite rules, mainly used to protect critical urls
+    # These rules are evaluated before the mapping rules
+    rewrites:
+      - comment: Security - deny access to product critical urls
+        rule: ^/admin/(.*)$               - [R=404,L]
+      - rule: ^/crx/(.*)$                 - [R=404,L]
+      - conditions:
+          - '%{REQUEST_URI} !^/system/sling/logout(.*)$'
+        rule: ^/system/(.*)$              - [R=404,L]
+      - rule: ^/api(.*)$                  - [R=404,L]
+      - rule: ^/apps/(.*)$                - [R=404,L]
+      - rule: ^/conf/(.*)$                - [R=404,L]
+      - rule: ^/bin/(.*)$                 - [R=404,L]
+      - rule: ^/services/(.*)$            - [R=404,L]
+      - rule: ^/tmp/(.*)$                 - [R=404,L]
+      - rule: ^/var/(.*)$                 - [R=404,L]
+      - rule: ^/dav/(.*)$                 - [R=404,L]
+      - rule: ^/jcr(.*)$                  - [R=404,L]
+      - rule: ^/_jcr(.*)$                 - [R=404,L]
+      - rule: ^/libs/wcm/(.*)$            - [R=404,L]
+      - rule: ^/libs/cq/(.*)$             - [R=404,L]
+      - rule: ^/var/linkchecker/(.*)$     - [R=404,L]
+      - rule: ^/etc/linkchecker/(.*)$     - [R=404,L]
+      - rule: ^/home/users/a/admin/(.*)$  - [R=404,L]
+      - rule: ^/apps/sling/(.*)$          - [R=404,L]
+      - rule: ^/dispatcher/invalidate(.*) - [R=404,L]
+
+    # POST access is forbidden, except for the followig URLs
+    uriAllowPostReqest:
```

Comment:
can this be integrated into the previous list of rewrites?

Reply:
There would be the possibility to integrate this into the same list, but the use-case is a little bit different.
The usecase of the rewriteIncludes are large lists (hundreds of lines) with legacy URLs that are shared over a list of tenants. with a format defined by the user These large lists would make the environment yaml hard to read. These files can then be static or dynamically rendered, depending on the use-case. 

The rewrite list is targeting the usecase were you have a short common list of rewrites which then can be extended using the _merge_. Since the list ist targeting securitly relevant rewrites that could result in 404 and 403 responses it makes it easier to debug when these rules are in the vhost itself.

Reply:
the question was if `uriAllowPostReqest` could/should be included in `rewrites` (not in `rewriteIncludes`).

Reply:
Sorry, I am wondering what I have read the first time :) 
From the technical perspective the `uriAllowPostRequest` list can be integrated in the rewrites list but I think we should keep them separated.
In the resulting vhost the readability is better when all URLs that are allowed for POST are at one place. 
If we put them in the rewrites list and add new allowed pathes with the "_merge_" construct we will have at least two places in the rendered list where we check for the POST request.